### PR TITLE
Fix multi subtree

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: lib/cfa/augeas_parser/writer.rb [hunk 1]
```diff
@@ -236,11 +236,13 @@ def add_entry(located_entry)
       # @see https://github.com/hercules-team/augeas/wiki/Path-expressions
       def set_new_value(path, located_entry)
         aug.set(path, located_entry.entry_value)
-        prefix = path[/(^.*)\[[^\]]*\]/, 1] || path
-        # we need to get new path as set can look like [last() + 1]
-        # which creates new entry and we do not want to add subtree to new
-        # entries
-        new_path = aug.match(prefix + "[last()]").first
+        # we need to get new path as path used in aug.set can look contain
```
Comment (imobachgs) [inline lib/cfa/augeas_parser/writer.rb]: "can look contain" -> "can contains" (?)

Comment (jreidinger) [inline lib/cfa/augeas_parser/writer.rb]: thanks, previously it was a bit different and I rewrite it and overlook it.

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Apr 25 10:49:03 UTC 2017 - jreidinger@suse.com
+
+- fix writting two elements in collection containing nested tree
+  ( also caused by fix for bsc#1023204)
```
Comment (imobachgs) [inline CHANGELOG]: np: just an extra space between the parenthesis and *also*.

**Comment by imobachgs** [comment]
It still LGTM :)

File: CHANGELOG [hunk 3]
```diff
@@ -1,7 +1,14 @@
 -------------------------------------------------------------------
+Tue Apr 25 10:49:03 UTC 2017 - jreidinger@suse.com
+
+- fix writting two elements in collection containing nested tree
```
Comment (mvidner) [inline CHANGELOG]: @jreidinger "writing"

Comment (jreidinger) [inline CHANGELOG]: ah, my favorite one. Will fix it in git

**Pull request merged**